### ISSUE TITLE
Remaining chapter time in topbar - adjust time depending on playback speed like timeline does?

### ISSUE THREAD

**Issue by xfzv** [tomasklaen/uosc#873]
Remaining chapter time in top bar was added with https://github.com/tomasklaen/uosc/commit/dc73278d657eec1b7236a4e7ad8466b6fd41f887

---

![Untitled](https://github.com/tomasklaen/uosc/assets/78810647/4895b20d-1258-489e-8efb-ecea6ee5d1db)

In the screenshot above:
With x2 playback speed, remaining time for the current chapter is 57:05.164 but the global remaining time is 40:45.560.

I guess remaining chapter time in top bar is always calculated based on 1x playback speed (unlike global remaining time in the timeline) considering it's always 57:05.164 despite adjusting the playback speed. The only difference is that 57:05.164 elapses faster or slower depending on playback speed. It would be nice if the actual remaining time was displayed instead, elapsing normally.

TL;DR: Any chance the remaining chapter time in top bar could adjust itself based on playback speed, like the timeline does? 

Thanks.
